osvr/RenderKit/osvr_display_configuration.cpp

```diff
@@ -783,6 +788,11 @@ OSVRDisplayConfiguration::getDistortionPolynomalBlue() const {
     return m_distortionPolynomialBlue;
 }
 
+int OSVR_RENDERMANAGER_EXPORT
```

Comment:
Why is it needed in most of the member function definitions but not in this one?